source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/registration/CardRendererRegistration.java

```diff
@@ -256,6 +257,14 @@ public IActionLayoutRenderer getOverflowActionLayoutRenderer()
         return m_overflowActionLayoutRenderer;
     }
 
+    public void registerTypeAheadParams(ITypeAheadCustomParams typeAheadCustomParams) {
```

Comment:
format to match the remainder of the document

Reply:
done